### PR TITLE
SQLAlchemy based example

### PR DESCRIPTION
**Pull request by nmusatti**
This is a simple example of how SQLAlchemy might be used to provide a backing store for a Connexion based application. It is based on Henning Jacobs's [Connexion Example REST Service](https://github.com/hjacobs/connexion-example).

### CODE REVIEW

**Comment by hjacobs** [comment]
Cool, thanks for the PR!

File: examples/sqlalchemy/orm.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            created=None):
+        if name is not None:
+            self.name = name
+        if animal_type is not None:
+            self.animal_type = animal_type
+        if created is not None:
+            self.created = created
+        
+    def dump(self):
+        return dict([(k,v) for k,v in self.__dict__.items() if k[0] != '_'])
```
Comment (jmcs) [inline examples/sqlalchemy/orm.py]: You can replace `self.__dict__` with `vars(self)` to avoid explicitly getting a dunder property.

Comment (hjacobs) [inline examples/sqlalchemy/orm.py]: and `k[0] != ...` is better replaced with `k.startswith(...)`

**Comment by nmusatti** [comment]
I applied the suggested changes and corrected import order, as signalled by the first test run.

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**